.github/workflows/release_linux.yml

```diff
@@ -37,16 +37,22 @@ jobs:
     steps:
     - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
       with:
-        persist-credentials: false
+        persist-credentials: true
```

Comment:
Add a comment on why this is needed?

Reply:
If we want to use the git command manually later, e.g. with “git config --global --add safe.directory /github/workspace”

Reply:
the default value is true

https://github.com/actions/checkout

```
    # Whether to configure the token or SSH key with the local git config
    # Default: true
    persist-credentials: ''
```